charts/opensearch/values.yaml

```diff
@@ -104,6 +104,10 @@ config:
 extraEnvs: []
 #  - name: MY_ENVIRONMENT_VAR
 #    value: the_value_goes_here
+# 2.12 onwards a custom strong password needs to be provided in order to setup demo admin user.
+# Cluster will not spin-up without this unless demo config install is disabled.
+#  - name: OPENSEARCH_INITIAL_ADMIN_PASSWORD
+#    value: <strong-password>
```

Comment:
I normally configure demo users(including `admin`) via `internal_users.yml` which password is hashed. Do I still need this env variable? Should it skip demo user initialization if `internal_users.yml` already contains admin password?

Reply:
> Do I still need this env variable?
Yes, for demo `admin` user setup this env variable will still be required.

> Should it skip demo user initialization if internal_users.yml already contained admin user?

If the security was already configured (i.e. opensearch.yml ) then the demo script will not execute. This variable is needed for fresh installs only.

Reply:
> If the security was already configured (i.e. opensearch.yml ) then the demo script will not execute. This variable is needed for fresh installs only.

If I just want the script to setup demo certificates for me as I configured the demo users myself, can I skip this env variable?

Reply:
no. If you run demo script, this variable is needed. It is not required only when running a setup without demo configuration.